src/i_oalsound.c

```diff
@@ -396,12 +396,14 @@ static void GetAttribs(boolean use_3d, ALCint *attribs)
         attribs[i++] = use_3d ? (snd_hrtf ? ALC_TRUE : ALC_FALSE) : ALC_FALSE;
     }
 
+#ifdef _WIN32
```

Comment:
It's not a Windows issue. The problem is that Ubuntu is still using libopenal_1.19.1 which doesn't have this extension in its headers. Please change this to e.g. `#ifdef ALC_OUTPUT_MODE_SOFT`.

Reply:
You're right, it looks like it wasn't added until [1.22](https://github.com/kcat/openal-soft/blob/e714c8fbca7bc4a03be308d43d6d6cd5e57914bc/ChangeLog#L113-L118).